app/controllers/pages_controller.rb

```diff
@@ -4,9 +4,11 @@ class PagesController < ApplicationController
   def inbox
     @filters = ["Inbox", "Reply later", "All"]
 
-    # TODO: This is a temporary solution to retrieve the last email server
-    @email_server = EmailServer.last
-    @emails = @email_server.retrieve_recent_emails if @email_server.present?
+    # TODO: This is a temporary solution to retrieve the last 20 emails
+    # TODO: Add a user setting to toggle email servers
+    @email_server = current_user.email_servers.first
+    @email_threads = @email_server.email_threads.includes(:messages).last(20)
```

Comment:
I think this is causing issues for the new users. When the user sign up then looks like they do not have any `email_server` at that stage and when landing on the inbox page directly after signup ends up in a error.

I'll put some checks around to fix it if that wasn't intentional

Reply:
Ah, yeah that's expected for now! Run `rails db:schema:load db:seed`. You should then be able to log in using `user@caley.to` and `password`. I still need to create an issue to fix user signups! We're working on getting the live email servers and mock email servers configured

Reply:
Cool, makes sense. Thanks!